### PR TITLE
Zjw dev

### PR DESCRIPTION
**Pull request by Zzjw**
Must Logged in before post comment!

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -32,9 +32,11 @@ <h1>Junwei Zhao</h1>
       <h2>COMMENT AREA</h2>
```
Comment (brianljt) [inline portfolio/src/main/webapp/index.html:32]: Nit: Add a button on main page to let user log out or switch user?

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -65,18 +65,50 @@ function createListElement(text) {
 }
 
 function getCommentList() {
-  fetch('/data').then(response => response.json()).then((comments) => {
+  fetch('/check').then(response => response.json()).then((value) => {
```
Comment (brianljt) [inline portfolio/src/main/webapp/script.js:68]: As main page will always load comment section, a user opens the main page will be redirected to do log in, even the user just want to view the page without leaving any comments. So my suggestion is to check log in status before `submit` button is hit and only force user to log in if it wants to leave any comment. Thoughts?

Comment (Zzjw) [reply portfolio/src/main/webapp/script.js]: Yes! Now I check log in status only before submit one's comment! Thanks a lot!

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (Zzjw) [reply portfolio/src/main/webapp/index.html:32]: Nice!

**Pull request merged**